WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingsViewController.swift

```diff
@@ -509,7 +509,7 @@ private extension NotificationSettingsViewController {
         var showBadge: Bool {
             switch self {
             case .blog:
-                return JetpackBrandingVisibility.all.enabled
+                return JetpackBrandingVisibility.all.enabled && JetpackBrandingCoordinator.shouldShowBannerOrBadgeForMainFeatures()
```

Comment:
[Suggestion]
```suggestion
                return JetpackBrandingVisibility.all.enabled && JetpackFeaturesRemovalCoordinator.jetpackFeaturesEnabled()
```
This way, there won't be need for adding `shouldShowBannerOrBadgeForMainFeatures`

Reply:
Thanks. Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20333/commits/5d2938bac30a51e202d9aa02b2440b83f7b347cd